go/api/go_api/session.py

```diff
@@ -12,7 +13,8 @@ class SessionStore(SessionBase):
     """
     def __init__(self, session_key=None):
         super(SessionStore, self).__init__(session_key)
-        self.session_manager = vumi_api().session_manager
+        self.session_manager = SessionManager(
+            get_redis_manager().sub_manager('session_manager'))
```

Comment:
I'm suddenly tempted to make the components of the VumiApi properties that
are only created when they're accessed. E.g.
```python

@property
def tpm(self):
    tpm = getattr(self, '_tpm', None)
    if tpm is None:
        tpm = self._tpm =
TagpoolManager(self.redis.sub_manager('tagpool_store'))
    return tpm
```
Not sure if this is a cool idea. We can probably avoid the boiler plate by
writing a `lazy_attribute` decorator that turns
```python
@lazy_property
def tpm(self):
    return TagpoolManager(self.redis.sub_manager('tagpool_store'))
```
into the property described previously.

If you like but don't feel like writing these, I'm happy to give them a go
in a separate PR.